python/pyarrow/tests/test_dataset.py

```diff
@@ -482,7 +482,7 @@ def test_filesystem_factory(mockfs, paths_or_selector):
     )
     assert options.partition_base_dir == 'subdir'
     assert options.ignore_prefixes == ['.', '_']
-    assert options.exclude_invalid_files is True
+    assert options.exclude_invalid_files is False
```

Comment:
the default changed here?

Reply:
Yes, validates require reading all files.